definitions/ext-rhost/definition.yml

```diff
@@ -0,0 +1,24 @@
+domain: EXT
+type: RHOST
+synthesis:
+  name: rfc460Hostname
+  identifier: rfc460Hostname
```

Comment:
One last question: are these values unique per account? what is the max length for them?

Reply:
It would be a unique IPV4 or FQDN and only report to a single account. The max length for IPV4 is 15 but the FQDN could be longer. I don't think any limits are set aside from the 4096 limit enforced by the Metrics API.

Reply:
It's probably a good idea then to set `encodeIdentifierInGUID: true` in that case (without this there is a chance that the identifier gets trimmed to the first ~50 characters.

Reply:
Thanks for the callout on that one, I updated the definition to include it.